ts/masters/modules/stock.src.ts

```diff
@@ -10,6 +10,7 @@
  * License: www.highcharts.com/license
  */
 'use strict';
+import Highcharts from '../highcharts.src.js';
```

Comment:
That needs to be taken from `'../../Core/Globals.js'` otherwise it bloats the product size.